docker/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM node:14.18.1-buster as build
+
+ARG COMMIT_REF
+ARG ENV
+ARG CHAIN
+ARG VEGA_URL
+ARG INFURA_ID
```

Comment:
Are these build-time args or run-time args?

`INFURA_ID` in particular looks like it needs to be a run-time arg.